src/aiai_eval/task.py

```diff
@@ -602,6 +615,14 @@ def _load_pytorch_model(
 
         return dict(model=model, tokenizer=tokenizer)
 
+    def _check_supertask(
+        self, architectures: Sequence[str], supertask: str, search_str: str
+    ):
```

Comment:
Missing docstring, including the `Raises:` block

Reply:
I am way to forgetful...